src_python/yaafelib/yaafe_extensions/yaafefeatures.py

```diff
@@ -941,7 +944,7 @@ def get_dataflow(cls, params, samplerate):
 #        stepSize = int(params['stepSize'])
 #        if (stepSize % (2**(nbdecim+nboct-1)))!=0:
 #            stepSize = stepSize - stepSize % (2**(nbdecim+nboct-1))
-#            print 'WARNING: adjust stepSize to %i to make it compatible with successive decimation'%stepSize
+#            print('WARNING: adjust stepSize to %i to make it compatible with successive decimation'%stepSize)
```

Comment:
Since you are modifying it ...
- add whitespace around `%` 
- split this long line